src/Carbon/Carbon.php

```diff
@@ -112,6 +112,11 @@ class Carbon extends DateTime
     const SECONDS_PER_MINUTE = 60;
 
     /**
+     * RFC7231 DateTime format.
+     */
+    const RFC7231 = 'D, d M Y H:i:s \G\M\T';
```

Comment:
For what its worth PHP merged in the same implementation today. https://github.com/php/php-src/commit/618d97962a8ac59d981c8307cc704790f46090c8